starter/source/model/sets/compute_connect_partelm.F

```diff
@@ -123,7 +125,8 @@ SUBROUTINE COMPUTE_CONNECT_PARTELM( IPARTS,IPARTQ,IPARTC,IPARTT,IPARTP,
       DO I=1,NUMSPH
         PART=IPARTSP(I)
         IND=INV_GROUP%INDPARTSPH(PART) + PART_COUNT(PART)
-        INV_GROUP%PARTSPH(IND)=I
+        !!INV_GROUP%PARTSPH(IND)=I
```

Comment:
Hi @mircea-altair 

I think no need of this comment , can you remove please ?

Thanks,
Sébastien

Reply:
Corrected!
Thanks